lib/commands/general.js

```diff
@@ -33,16 +33,50 @@ commands.getStrings = async function (language) {
     log.info(`No language specified, returning strings for: ${language}`);
   }
 
+  // Clients require the resulting mapping to have both keys
+  // and values of type string
+  const preprocessStringsMap = (mapping) => {
```

Comment:
I prefer `function preprocessStringsMap () { ... }` so the name of the function shows up in the callstack if there's an error.